### PR TITLE
Rename Ingest to Imported and Audited to Processed.

### PR DESCRIPTION
**Pull request by manthey**

### CODE REVIEW

File: nci_seer/web_client/templates/itemView.pug [hunk 1]
```diff
@@ -8,7 +8,7 @@
     finished: ['quarantine']
   };
   var buttons = {
-    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the Audited location'},
+    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the processed folder'},
```
Comment (mgrauer) [inline nci_seer/web_client/templates/itemView.pug:11]: sorry to pick nits, could you use consistent capitalization in these titles? currently only the `finish` has the name of the target location capitalized and the others do not, i.e. `'Move this image to the Finished location'`

File: nci_seer/web_client/templates/itemView.pug [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
     reject: {name: 'Reject', title: 'Select this if redaction is insufficient.  The image will be moved to the rejected folder'},
     quarantine: {name: 'Quarantine', title: 'Move this image to the quarantine folder for redaction'},
     unquarantine: {name: 'Undo Quarantine', title: 'Move this image back to its location before quarantining'},
```
Comment (mgrauer) [inline nci_seer/web_client/templates/itemView.pug:14]: where does `back to its location` mean?

File: nci_seer/web_client/templates/itemView.pug (hunk 1 above)
Comment (manthey) [reply nci_seer/web_client/templates/itemView.pug:11]: Will do.  I appreciate the feedback.

File: nci_seer/web_client/templates/itemView.pug (hunk 2 above)
Comment (manthey) [reply nci_seer/web_client/templates/itemView.pug:14]: You can quarantine an item from anywhere in the system except the quarantine folder.  This puts it back to its location before quarantining (it undoes the quarantine action).

Comment (manthey) [reply nci_seer/web_client/templates/itemView.pug:14]: Any suggestions on clearer wording?

Comment (mgrauer) [reply nci_seer/web_client/templates/itemView.pug:14]: would they know what the previous location is? that may be a separate usability issue than this language, but just something to think about


maybe:

`Move this image back to its previous location before it was quarantined`

Comment (manthey) [reply nci_seer/web_client/templates/itemView.pug:14]: The original location is stored in the item's metadata as a folder ID.  We could do a REST request and look it up -- but I think that is a feature that I'd rather see moved upstream to HistomicsUI and left for a later time.

**Pull request merged**